data/json/items/gun/5x50.json

```diff
@@ -21,7 +21,11 @@
     "range" : 10,
     "durability": 9,
     "barrel_length" : 1,
-    "burst": 10,
+    "modes": [
+      [ "DEFAULT", "semi-auto", 1 ],
+      [ "BURST", "3 rd.", 3 ],
```

Comment:
I thought these would be improved by burst fire, now that firemodes are possible.